tools/ssl/Makefile

```diff
@@ -1,41 +1,45 @@
-CA_SUB  = /C=PL/ST=Malopolska/L=Krakow/CN=MongooseIM Fake CA
-SRV_SUB = /C=PL/ST=Malopolska/L=Krakow/CN=MongooseIM Server
+all: mongooseim/server.pem mongooseim/cert.pem \
+  mongooseim/key.pem mongooseim/dh_server.pem \
+  mongooseim/pubkey.pem mongooseim/privkey.pem \
+  ca/cacert.pem ca/cakey.pem \
 
-all: ca/cacert.pem fake_cert.pem fake_pubkey.pem fake_privkey.pem fake_server.pem fake_dh_server.pem
+%/cacert.pem: openssl-%.cnf
```

Comment:
This rule creates both `%/cacert.pem` and `ca/cacert.key`. I propose:

* Set `%/cacert.pem %/cacert.key` as targets (because rule creates both)
* Remove `default_keyfile` from `openssl-ca.cnf` (because of hardcoded path)
* Call `openssl req -x509 -config $< -newkey rsa:2048 -sha256 -nodes -out $(@D)/cacert.pem -outkey $(@D)/cakey.pem -outform PEM`

This has the additional benefit of being more explicit (I had to do some thinking on the `%/cakey.pem: %/cacert.pem;` rule